src/quo/components/community/style.cljs

```diff
@@ -133,25 +133,6 @@
                       colors/neutral-80
```

Comment:
hmm this file still seems in the wrong place - 
src/quo/components/community/style.cljs is like at the folder level, can we move it to sit beside it's corresponding `view` file?

Reply:
this seems to be an old ns. This file holds styles for a bunch of components from community, so better have a separate PR to clean it up

Reply:
ah sorry I thought it was associated with src/quo/components/community/channel_actions/view.cljs.

yeah separate issue to fix sounds good 👍